### PR TITLE
MNT: rerender for macosx compat

### LINKED ISSUE #14: Fixing OS X compatibility
Please [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) and bump the build number. This is needed to fix an issue where Travis CI bumped us from OS X version 10.9 to 10.11.

xref: https://github.com/conda-forge/conda-forge.github.io/issues/249

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
Closes #14

### CODE REVIEW

**Comment by jakirkham** [comment]
Did you [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) with `conda-smithy` version `1.3.2`? Seems to be missing a critical piece.

**Comment by jorisvandenbossche** [comment]
I thought so, but apparently not, still 1.3.1. Now it should be good. Thanks for noticing!

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
```
Comment (jakirkham) [inline .travis.yml]: 👍

**Comment by jakirkham** [comment]
Yep, looks better now. Thanks for fixing it. 😄

**Pull request merged**